app/imageswap-init/imageswap-init.py

```diff
@@ -621,8 +648,8 @@ def init_tls_pair(namespace):
             sys.exit(1)
 
     configuration = client.Configuration()
-    core_api = client.CoreV1Api(client.ApiClient(configuration))
-    certificates_api = client.CertificatesV1beta1Api(client.ApiClient(configuration))
+    core_api = client.CoreV1Api(client.ApiClient())
```

Comment:
Is there q reason the `configuration` input was dropped to `client.ApiClient()`

Reply:
Now I don't know the specific reason for it but at the time I tried to make things work with the new API, I've found in some documentation that I need to do like this once I don't use any non-default param.